pilot/pkg/kube/inject/mesh.go

```diff
@@ -118,6 +118,8 @@ containers:
   - [[ .ProxyConfig.StatsdUdpAddress ]]
   - --proxyAdminPort
   - [[ .ProxyConfig.ProxyAdminPort ]]
+  - --concurrency
+  - [[ .ProxyConfig.Concurrency ]]
```

Comment:
@costinm Should this stay `.ProxyConfig.Concurrency` in kube-inject? or should I completely remove it from here?

Reply:
This file is only used by some tests.